pkg/ebpf/windows_utils.go

```diff
@@ -0,0 +1,8 @@
+// +build windows
+
+package ebpf
+
+
+func verifyOSVersion(kernelCode uint32, platform string, exclusionList []string) (bool, string) {
+	return true, ""
```

Comment:
Add a TODO that we this still needs to be fleshed out for the windows versions we support + that we should also probably remove `kernelCode` from it's parameters.